README.md

```diff
@@ -36,51 +37,51 @@ All the translations for this repo will be listed below:
 5. **[== vs === vs typeof](#5--vs--vs-typeof)**
-6. **[Function Scope, Block Scope and Lexical Scope](#6-function-scope-block-scope-and-lexical-scope)**
-7. **[Expression vs Statement](#7-expression-vs-statement)**
-8. **[IIFE, Modules and Namespaces](#8-iife-modules-and-namespaces)**
-9. **[Message Queue and Event Loop](#9-message-queue-and-event-loop)**
-10. **[setTimeout, setInterval and requestAnimationFrame](#10-settimeout-setinterval-and-requestanimationframe)**
-11. **[JavaScript Engines](#11-javascript-engines)**
-12. **[Bitwise Operators, Type Arrays and Array Buffers](#12-bitwise-operators-type-arrays-and-array-buffers)**
-13. **[DOM and Layout Trees](#13-dom-and-layout-trees)**
-14. **[Factories and Classes](#14-factories-and-classes)**
-15. **[this, call, apply and bind](#15-this-call-apply-and-bind)**
-16. **[new, Constructor, instanceof and Instances](#16-new-constructor-instanceof-and-instances)**
-17. **[Prototype Inheritance and Prototype Chain](#17-prototype-inheritance-and-prototype-chain)**
-18. **[Object.create and Object.assign](#18-objectcreate-and-objectassign)**
+6. **[Zakres Funkcji, Zakres Bloku i Zakres Leksykalny](#6-function-scope-block-scope-and-lexical-scope)**
+7. **[Wyrażenie vs Instrukcja](#7-expression-vs-statement)**
+8. **[IIFE, Moduły i Przestrzenie Nazw](#8-iife-modules-and-namespaces)**
+9. **[Message Queue i Event Loop](#9-message-queue-and-event-loop)**
+10. **[setTimeout, setInterval i requestAnimationFrame](#10-settimeout-setinterval-and-requestanimationframe)**
+11. **[Silniki JavaScript](#11-javascript-engines)**
+12. **[Operatory Bitowe, Type Arrays i Array Buffers](#12-bitwise-operators-type-arrays-and-array-buffers)**
+13. **[DOM i Layout Trees](#13-dom-and-layout-trees)**
+14. **[Fabryki i Klasy](#14-factories-and-classes)**
+15. **[this, call, apply i bind](#15-this-call-apply-and-bind)**
+16. **[new, Constructor, instanceof oraz Instances](#16-new-constructor-instanceof-and-instances)**
+17. **[Dziedziczenie Prototypowe i Łańcuch Prototypów](#17-prototype-inheritance-and-prototype-chain)**
+18. **[Object.create i Object.assign](#18-objectcreate-and-objectassign)**
 19. **[map, reduce, filter](#19-map-reduce-filter)**
-20. **[Pure Functions, Side Effects and State Mutation](#20-pure-functions-side-effects-and-state-mutation)**
-21. **[Closures](#21-closures)**
-22. **[High Order Functions](#22-high-order-functions)**
-23. **[Recursion](#23-recursion)**
-24. **[Collections and Generators](#24-collections-and-generators)**
-25. **[Promises](#25-promises)**
+20. **[Funkcje Czyste, Efekty Uboczne i Mutacje Stanu](#20-pure-functions-side-effects-and-state-mutation)**
+21. **[Domknięcia](#21-closures)**
+22. **[Funkcje Wyższego Rzędu](#22-high-order-functions)**
+23. **[Rekursja/Rekurencja](#23-recursion)**
+24. **[Generatory i Kolekcje](#24-collections-and-generators)**
+25. **[Obietnice](#25-promises)**
```

Comment:
I am not sure if you should translate that, it makes me confused. If yes, please add the original name in the brackets.

Reply:
Dzięki serdeczne, też tak myślałem.